src/test/kotlin/CheckersModelTest.kt

```diff
@@ -55,18 +188,182 @@ class TestCanMove {
             "Wrong square returned: eaten checker expected")
         assertNull(m.canMove(BaseTurn(Color.WHITE, Pair(5, 0), Pair(4, 1))), "Player moved wrong checker")
     }
+}
+
+class CommonLogicTest {
+
+    private val board = CheckersBoard(8)
+    private val model = CheckersModel()
+
+    private fun wrongMove(turn: BaseTurn) {
+        assertNull(model.canMove(turn), "Can make turn when it is wrong")
+    }
+
+    private fun rightMove(turn: BaseTurn) {
+        assertNotNull(model.canMove(turn), "Can't make turn when it is right")
```

Comment:
I believe that word "correct" is more suitable here

Reply:
Fixed